Interface conversion error while live loading exported JSON data

### What version of Dgraph are you using?
`master`

### Steps to reproduce the issue (command/config used to run Dgraph).

- [Exporting database](https://docs.dgraph.io/deploy/#export-database) in JSON. 

```bash
$ curl 'localhost:8080/admin/export?format=json'
{"code": "Success", "message": "Export completed."}
```
- Fast load the data using [Live Loader](https://docs.dgraph.io/deploy/#live-loader).

```bash
$ dgraph live -f g01.json.gz -s g01.schema --alpha=localhost:9080 --zero=localhost:5080
...

Running transaction with dgraph endpoint: localhost:9080

Processing schema file "g01.schema"
Processed schema file "g01.schema"

Found 1 data file(s) to process
Processing data file "g01.json.gz"
panic: interface conversion: interface {} is *time.Time, not time.Time

goroutine 45 [running]:
github.com/dgraph-io/dgraph/tok.HourTokenizer.Tokens(0x1818880, 0xc0027920a0, 0x10, 0xc004074020, 0xa, 0xc00328c003, 0xc000131880)
	/home/prashant/go/src/github.com/dgraph-io/dgraph/tok/tok.go:262 +0x3ca
github.com/dgraph-io/dgraph/tok.BuildTokens(0x1818880, 0xc0027920a0, 0x1a230e0, 0x261d298, 0x1a230e0, 0x261d298, 0x0, 0x1, 0xc004074010)
	/home/prashant/go/src/github.com/dgraph-io/dgraph/tok/tok.go:104 +0x63
github.com/dgraph-io/dgraph/dgraph/cmd/live.(*loader).conflictKeysForNQuad(0xc0002e8000, 0xc00348c510, 0x3e8, 0xc004072000, 0xc000045380, 0x0, 0xc00009db00)
	/home/prashant/go/src/github.com/dgraph-io/dgraph/dgraph/cmd/live/batch.go:301 +0x448
github.com/dgraph-io/dgraph/dgraph/cmd/live.(*loader).conflictKeysForReq(0xc0002e8000, 0xc000459f88, 0xc0007e2001, 0x7d0, 0x800)
	/home/prashant/go/src/github.com/dgraph-io/dgraph/dgraph/cmd/live/batch.go:322 +0xe9
github.com/dgraph-io/dgraph/dgraph/cmd/live.(*loader).makeRequests(0xc0002e8000)
	/home/prashant/go/src/github.com/dgraph-io/dgraph/dgraph/cmd/live/batch.go:384 +0x1db
created by github.com/dgraph-io/dgraph/dgraph/cmd/live.setup
	/home/prashant/go/src/github.com/dgraph-io/dgraph/dgraph/cmd/live/run.go:356 +0x2f0
```

### Expected behaviour and actual result.

This issue is seen only while live loading **JSON** data and not for **RDFs**.

Also, I was able to **bulk load** the exported data in both JSON and RDFs without any problem.

*Note: Dataset used here was from [JS Flock](https://github.com/dgraph-io/flock/tree/master/js)*.

--- a/dgraph/cmd/live/batch.go
+++ b/dgraph/cmd/live/batch.go
@@ -301,7 +301,11 @@ func (l *loader) conflictKeysForNQuad(nq *api.NQuad) ([]uint64, error) {
 			Tid:   types.TypeID(de.GetValueType()),
 			Value: de.GetValue(),
 		}
-
+		// If the value type is not already set according to the schema, we set it to string and
+		// then convert it to the type as declared in the schema.
+		if storageVal.Tid != pred.ValueType {
+			storageVal.Tid = types.StringID
+		}
 		schemaVal, err := types.Convert(storageVal, types.TypeID(pred.ValueType))
 		if err != nil {
 			errs = append(errs, err.Error())

--- a/dgraph/cmd/live/batch.go
+++ b/dgraph/cmd/live/batch.go
@@ -301,7 +301,7 @@ func (l *loader) conflictKeysForNQuad(nq *api.NQuad) ([]uint64, error) {
 			Tid:   types.TypeID(de.GetValueType()),
 			Value: de.GetValue(),
 		}
-		// If the value type is not already set according to the schema, we set it to string and
+		// If the value type is not already set according to the schema, set it to string and
 		// then convert it to the type as declared in the schema.
 		if storageVal.Tid != pred.ValueType {
 			storageVal.Tid = types.StringID

